vertx-hazelcast/src/main/java/module-info.java

```diff
@@ -25,11 +25,11 @@
   requires io.vertx.core;
   requires io.vertx.core.logging;
   requires com.hazelcast.core;
-    requires io.netty.common;
+  requires io.netty.common;
 
-    exports io.vertx.spi.cluster.hazelcast;
+  exports io.vertx.spi.cluster.hazelcast;
   exports io.vertx.spi.cluster.hazelcast.spi;
-  exports io.vertx.spi.cluster.hazelcast.impl to io.vertx.clustermanager.hazelcast.tests, com.hazelcast.core;
```

Comment:
add a comment about the reason of this export so other committers will understand the reason